hack/testdata/recursive/deployment/deployment/busybox-broken.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: extensions/v1beta1
+ind: Deployment
+metadata:
+  name: nginx2-deployment
+  labels:
+    app: nginx2-deployment
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: nginx2
+    spec:
+      containers:
+      - name: nginx
+        image: gcr.io/google-containers/nginx:1.7.9
```

Comment:
the `kind` key has an intentional typo to issue a simple error upon usage with the subcommands